desci-server/src/services/node.ts

```diff
@@ -121,35 +122,31 @@ export const countPublishedNodesInRange = async (range: { from: Date; to: Date }
   return publishes;
 };
 
-export const getNodeDetails = async (nodeUuid: string) => {
+const NODE_DETAILS_CACHE_KEY = `NODE_DETAILS_CACHE_KEY`;
+
+export interface NodeDetails {
+  versions?: number;
+  publishedDate?: Date;
+  manifestUrl?: string;
+  dpid?: number;
+  dpidAlias?: number;
+  authors?: any[];
+}
+
+export const getNodeDetails = async (discovery: Partial<Node>) => {
+  if (!discovery) return {};
+
   const logger = parentLogger.child({ module: 'getNodeDetails' });
-  const uuid = ensureUuidEndsWithDot(nodeUuid);
 
-  const discovery = await prisma.node.findFirst({
-    where: {
-      uuid,
-      isDeleted: false,
-    },
-    select: {
-      id: true,
-      manifestUrl: true,
-      ownerId: true,
-      uuid: true,
-      title: true,
-      NodeCover: true,
-      dpidAlias: true,
-      legacyDpid: true,
-      manifestDocumentId: true,
-    },
-  });
+  const cacheKey = `${NODE_DETAILS_CACHE_KEY}_${discovery.id}`;
```

Comment:
this will cache v1 and never return v2 if updated

Reply:
fixed